src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  &__title {
+    padding-inline: 16px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+
+    position: absolute;
+    top: 206px;
```

Comment:
Using absolute positioning for the title might cause issues with responsiveness. Consider using relative positioning or a layout model like flexbox or grid to ensure the title adjusts well on different screen sizes.